packages/desktop-client/src/components/budget/BudgetTable.jsx

```diff
@@ -29,12 +30,36 @@ export function BudgetTable(props) {
     onBudgetAction,
   } = props;
 
+  const budgetCategoriesRef = useRef();
   const { grouped: categoryGroups } = useCategories();
   const [collapsedGroupIds = [], setCollapsedGroupIdsPref] =
     useLocalPref('budget.collapsed');
   const [showHiddenCategories, setShowHiddenCategoriesPef] = useLocalPref(
     'budget.showHiddenCategories',
   );
+
+  const scrollToPosition = () => {
+    sessionStorage.setItem(
```

Comment:
Without a ton more code for something simplistic, sessionstorage is by far the easiest way. We are going from one component to another, and recalling from a higher component than other. To do this would be alot more complex...

Reply:
I'll give it a shot tomorrow.